test/integration/connect/envoy/run-tests.windows.sh

```diff
@@ -332,14 +332,7 @@ function verify {
 
   # need to tell the PID 1 inside of the container that it won't be actual PID
   # 1 because we're using --pid=host so we use TINI_SUBREAPER
-  if docker.exe run --name envoy_verify-${CLUSTER}_1 -t \
-    -e TINI_SUBREAPER=1 \
-    -e ENVOY_VERSION \
-    $WORKDIR_SNIPPET \
-    --pid=host \
-    $(network_snippet $CLUSTER) \
-    bats-verify \
-    --pretty ${CLUSTER}/bats ; then
+  if docker.exe exec -i envoy_consul-${CLUSTER}_1 bash -c "/c/bats/bin/bats /c/workdir/${CLUSTER}/bats/verify.bats" ; then
```

Comment:
Let's use the `SINGLE_CONTAINER_BASE_NAME` variable declared in the `line 12` to be consistent with the others docker exec commands and any change is easier